pkg/utils/utils.go

```diff
@@ -22,6 +22,16 @@ func Max(x, y int) int {
 	return y
 }
 
+// Clamp returns a value x restricted between min and max
+func Clamp(x int, min int, max int) int {
```

Comment:
Just checking: is there a TestClamp in lazygit that we want to bring across?

Reply:
There isn't ([file](https://github.com/gusandrioli/lazygit/blob/master/pkg/utils/utils_test.go)). But i can add one quickly in a bit :).